great_expectations/datasource/fluent/interfaces.py

```diff
@@ -228,6 +230,7 @@ def build_batch_request(
             batch_slice: A python slice that can be used to limit the sorted batches by index.
                 e.g. `batch_slice = "[-5:]"` will request only the last 5 batches after the options filter is applied.
             partitioner: A Partitioner used to narrow the data returned from the asset.
+            batching_regex: A Regular Expression used to build batches in some types of Assets.
```

Comment:
s/in some of/in path based/

Reply:
[66023b6](https://github.com/great-expectations/great_expectations/pull/9721/commits/66023b690e233a582597322e30a0164f716ccf80)